kolibri/plugins/device/assets/src/views/ManageContentPage/SelectNetworkAddressModal/SelectAddressForm.vue

```diff
@@ -173,22 +203,47 @@
           this.selectedAddressId = '';
         }
       },
-      removeAddress(id) {
+      removeSavedAddress(id) {
         this.stage = this.Stages.DELETING_ADDRESS;
         return deleteAddress(id)
           .then(() => {
-            this.addresses = this.addresses.filter(a => a.id !== id);
-            this.resetSelectedAddress(this.addresses);
+            this.savedAddresses = this.savedAddresses.filter(a => a.id !== id);
+            this.resetSelectedAddress(this.savedAddresses);
             this.stage = this.Stages.DELETING_SUCCESSFUL;
             this.$emit('removed_address');
           })
           .catch(() => {
             this.stage = this.Stages.DELETING_FAILED;
           });
       },
+
+      discoverPeers() {
+        this.$parent.$emit('started_peer_discovery');
+        return fetchDevices(this.isImportingMore ? this.transferredChannel.id : '')
+          .then(devices => {
+            this.$parent.$emit('finished_peer_discovery');
+            this.devices = devices;
+          })
+          .catch(() => {
+            this.$parent.$emit('peer_discovery_failed');
+          });
+      },
+
+      startDiscoveryPolling() {
+        if (!this.intervalId) {
+          this.intervalId = setInterval(this.discoverPeers, 5000);
+        }
+      },
+
+      stopDiscoveryPolling() {
+        if (this.intervalId) {
+          this.intervalId = clearInterval(this.intervalId);
+        }
+      },
+
```

Comment:
same comment as in another PR here: https://github.com/learningequality/kolibri/pull/5631#discussion_r337258258